IdentityCore/src/oauth2/token/MSIDBaseToken.m

```diff
@@ -139,9 +139,21 @@ - (instancetype)initWithTokenCacheItem:(MSIDCredentialCacheItem *)tokenCacheItem
             MSID_LOG_WARN(nil, @"Trying to initialize token when missing clientId field");
             return nil;
         }
-        
-        _additionalServerInfo = tokenCacheItem.additionalInfo;
 
+        NSMutableDictionary* serverInfo = [NSMutableDictionary new];
```

Comment:
nit: would it make it easier if we handled those fields similarly between cache items and tokens? I'm fine getting rid of additionalInfo outside of legacy cache item serialization and additional fields coming from server that we don't understand.

Reply:
Seems like a good idea. Updated.